script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt

```diff
@@ -679,22 +774,25 @@ When did you get so confident that you could declare victory in this situation?"
 [sha:992d10ee8e0fcc5ad2d3e4cc59f9772c7220fc3c]{
 -- Page 140, Offset 38722.
 -- 「もちろん。おかげで安心できました。
-"I know. Thanks to that, I'm able to feel at ease.
+"Yeah, I know.
+ I feel a lot safer thanks to you.
 }
 [sha:4700720c09cdfa6b337cc4ad07879f207909c8b6]{
 -- Page 140, Offset 38723.
 -- 大丈夫、ロアには負けないって言ったでしょう。俺が勝てないのは先輩だけです」
-It's okay. I told you that I won't lose to Roa, didn't I? The only one I can't beat is you."
+And don't worry――I told you I wouldn't lose to Roa, didn't I?
+ The only person I can't win against is you."
 }
 [sha:e97cfef119c8bdaf71be234c702c611afe2a5aa4]{
 -- Page 141, Offset 38724.
 -- ありったけの信頼をこめて、笑って送り出す。
-Mustering up all the faith that I have, I pass over a smile.
+I give her a farewell smile laden with all the trust I feel.
```

Comment:
Wow, what an awful day to be a translator. I hate Japanese.
I feel like it needs really drastic changes, but no inspiration hit.

Reply:
>I send her off with a smile, doing my utmost to show just how much faith I have in her.

Best I can do.